spec/controllers/stories_controller_spec.rb

```diff
@@ -146,4 +146,69 @@
       end
     end
   end
+
+  describe '#UPDATE' do
+    let(:attr) do
+      {
+        url: 'http://newexample.com/',
```

Comment:
Please use `example.com` - it's set aside for tests and examples. `newexample.com` is an SEO/marketing firm with an expired SSL cert.

Reply:
done

Reply:
Done.